graphdatascience/system/system_endpoints.py

```diff
@@ -16,6 +16,48 @@ def sysInfo(self) -> "Series[Any]":
         return self._query_runner.run_query(query).squeeze()  # type: ignore
 
 
+class DefaultAndLimitsProcRunner(CallerBase, UncallableNamespace, IllegalAttrChecker):
+    def set(self, key: str, value: Any, username: Optional[str] = None) -> None:
+        self._namespace += ".set"
+
+        params = {"key": key, "value": value}
+
+        # Checking for explicit None as '' is a valid user
+        if username is not None:
+            query = f"CALL {self._namespace}($key, $value, $username)"
+            params["username"] = username
+        else:
+            query = f"CALL {self._namespace}($key, $value)"
+
+        self._query_runner.run_query(query, params)
+
+    def list(self, key: Optional[str] = None, username: Optional[str] = None) -> DataFrame:
+        self._namespace += ".list"
+
+        config: Dict[str, Any] = {}
+
+        if key:
+            config["key"] = key
+        # Checking for explicit None as '' is a valid user
+        if username is not None:
+            config["username"] = username
+
+        query = f"CALL {self._namespace}($config)"
+        params = {"config": config}
+
+        return self._query_runner.run_query(query, params)
+
+
+class DefaultsAndLimitsEndPoints(CallerBase):
```

Comment:
This also needs to inherit from `UncallableNamespace` and `IllegalAttrChecker` for nicer error handling